tslint.json

```diff
@@ -128,7 +129,22 @@
     ],
     "triple-equals": [true, "allow-null-check"],
     "use-isnan": true,
-    "variable-name": [true, "ban-keywords", "check-format", "allow-pascal-case"]
+    "variable-name": [true, "ban-keywords", "check-format", "allow-pascal-case"],
+    "jsx-alignment": false,
+    "jsx-ban-elements": false,
+    "jsx-ban-props": false,
+    "jsx-boolean-value": false,
+    "jsx-curly-spacing": false,
+    "jsx-equals-spacing": false,
+    "jsx-key": false,
+    "jsx-no-bind": false,
+    "jsx-no-lambda": true,
+    "jsx-no-multiline-js": false,
+    "jsx-no-string-ref": false,
+    "jsx-use-translation-function": false,
+    "jsx-self-close": false,
+    "jsx-space-before-trailing-slash": false,
+    "jsx-wrap-multiline": false
```

Comment:
I disabled those rules since I only wanted the `jsx-no-lambda`, but they're all enabled by default when you extend the preset.

Enabling them might require some fixes across the codebase.